index.css

```diff
@@ -69,3 +69,216 @@
   font-size: 16px;
   line-height: 24px;
 }
+
+.achivement {
+  width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.recent {
+  width: 327px;
+  height: 83px;
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 40;
+  line-height: 52px;
+  color: #1c1a19;
+}
+
+.works {
+  display: grid;
+}
+
+.storie h2 {
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.storie p {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.tools {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+  padding: 0;
+  margin: 0;
+  margin-bottom: 10px;
+}
+
+.tools li {
+  list-style: none;
+  padding: 10px;
+  margin-right: 10px;
+  border: 1px solid black;
+}
+
+.button {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 3%;
+  text-align: center;
+  background-color: #ff6b00;
+  color: #fff;
+  border: 2px solid #ff6b00;
+}
+
+.button:hover {
+  color: #ff6b00;
+  background-color: #fff;
+  border: 2px solid #ff6b00;
+}
+
+.button:active {
+  color: #fff;
+  background-color: #ff6b00;
+}
+
+.button:disabled {
+  background-color: #5b5957;
+}
```

Comment:
- [ ] Good job implementing the different states of the buttons, but you should also use the correct colors provided by the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=37%3A2154), please click on each button and check in the inspect section which color is used.

This is a quick guide to show you how, in case you don't know:
![image](https://user-images.githubusercontent.com/20567503/176929637-8d5d8af0-ba8d-44fd-a051-bad679755c1a.png)